README.md

```diff
@@ -1,9 +1,126 @@
 # Wavefront Dropwizard Metrics SDK [![build status][ci-img]][ci] [![Released Version][maven-img]][maven]
 
-The Wavefront by VMware Dropwizard Metrics SDK for Java is a library that supports reporting [Dropwizard metrics and histograms](https://metrics.dropwizard.io) to Wavefront.
-
-## Maven
-If you are using Maven, add the following maven dependency to your pom.xml:
+## Table of Content
+* [Prerequisites](#Prerequisites)
+* [Set Up a DropwizardMetricsReporter](#set-up-a-dropwizardmetricsreporter)
+  * [Step 1: Create a Builder for a DropwizardMetricsReporter](#step-1-create-a-builder-for-a-dropwizardmetricsreporter)
+  * [Step 2: Configure the DropwizardMetricsReporter (Optional)](#step-2-configure-the-dropwizardmetricsreporter-optional)
+  * [Step 3: Set Up a WavefrontSender](#step-3-set-up-a-wavefrontsender)
+  * [Step 4: Create a DropwizardMetricsReporter](#step-4-create-a-dropwizardmetricsreporter)
+* [Start the DropwizardMetricsReporter](#start-the-dropwizardmetricsreporter)
+* [Stop the DropwizardMetricsReporter](#stop-the-dropwizardmetricsreporter)
+* [Types of Data You Can Report to Wavefront](#types-of-data-you-can-report-to-wavefront)
+* [Monitoring the SDK](#monitoring-the-sdk)
+* [License](#License)
+* [How to Contribute](#How-to-Contribute)
+
+# Welcome to the Wavefront Dropwizard Metrics SDK
+
+The Wavefront by Tanzu Observability by Wavefront Dropwizard Metrics SDK for Java is a library that supports reporting [Dropwizard metrics and histograms](https://metrics.dropwizard.io) to Wavefront.
```

Comment:
The Wavefront by Tanzu Observability by Wavefront...
RK>>The Tanzu Observability by Wavefront ...